mods/src/config.cc

```diff
@@ -274,15 +281,62 @@ void parse_config_shortcut(toml::table config, toml::table& new_config, std::str
   spdlog::info("shortcut value {}.{} value: {}", section, item, shortcut);
 }
 
+std::string_view Config::Filename()
+{
+  static std::string configFile = "";
+
+  if (configFile.empty()) {
+#if _WIN32
+    // Get the command line
+    LPCWSTR cmdLine = GetCommandLineW();
+
+    // Parse command line into individual arguments
+    int     argc;
+    LPWSTR* argv = CommandLineToArgvW(cmdLine, &argc);
+
+    // If we have some arguments, lets see what we got
+    std::wstring argValue;
+    if (argv != nullptr) {
+      // Output the arguments (for example purposes, we'll just print them)
+      for (int i = 0; i < argc - 1; ++i) {
+        if (std::wstring(argv[i]) == L"-ccm" && i + 1 < argc) {
+          // Found "-ccm", so take the next argument as the value
+          argValue = argv[i + 1];
+          break;
+        }
+      }
+
+      if (!argValue.empty()) {
+        configFile.assign(argValue.begin(), argValue.end());
```

Comment:
This is not doing proper conversions, files names should be UTF-8 encoded, this doesn't convert to UTF-8 and will result in garbage for non-ascii filenames

Reply:
thanks, I had a feeling that might be the case.